routes/url-api-routes.js

```diff
@@ -5,7 +5,7 @@ module.exports = function(app) {
 
 // GET route to get session ID
   app.get("/api/urls/:pw", function(req, res) {
-    db.user.findAll({
+    db.Url.findAll({
```

Comment:
I can change the models if you'd like, just let me know which way you'd like to go